### PR TITLE
Dotson contrib

### PR DESCRIPTION
**Pull request by samgdotson**
Adds images and slides for `dotson.tex`. Both annsa and reactor control project are included under the title of "Machine Learning Applications in Nuclear Engineering."

### CODE REVIEW

**Comment by smpark7** [comment]
Hey Sam!

You don't need to include a preamble in dotson.tex because it is already included in the arfc-pres.tex file. The arfc-pres.tex file imports your dotson.tex so that everyone's slides are imported into the one presentation pdf file (arfc-pres.pdf). 

Also, a more descriptive figure caption would be helpful for the ANNSA CNN figure.

**Comment by samgdotson** [comment]
> You don't need to include a preamble in dotson.tex because it is already included in the arfc-pres.tex file. The arfc-pres.tex file imports your dotson.tex so that everyone's slides are imported into the one presentation pdf file (arfc-pres.pdf).

I left the `\graphicspath{}` because `make` was throwing a fit when I commented it out.

File: dotson.tex [hunk 1]
```diff
@@ -0,0 +1,84 @@
+% needed to create images without issue
+\graphicspath{{./images/}}
```
Comment (smpark7) [inline dotson.tex]:
```suggestion
```

File: dotson.tex [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+	\begin{frame}
+		\frametitle{Table of Contents}
+		\tableofcontents
+	\end{frame}
+
+	\section{ANNSA}
```
Comment (smpark7) [inline dotson.tex]:
```suggestion
```
This probably falls under the software section

File: dotson.tex [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+	\begin{frame}
+		\frametitle{$\texttt{ANNSA}$ - Extensions and Future Work}
+		\begin{enumerate}
+			\item Algorithm Improvements
+			\item Different Detectors
+			\item More complicated isotopics
+		\end{enumerate}
+	\end{frame}
+
+	\section{CAIRO}
```
Comment (smpark7) [inline dotson.tex]:
```suggestion
```
This falls under software section too?

**Review by smpark7 — CHANGES_REQUESTED** [review]

**Comment by samgdotson** [comment]
closes issue #11

**Pull request merged**